src/net/RTCP/RTCPSession.cs

```diff
@@ -220,7 +225,8 @@ public void RecordRtpPacketReceived(RTPPacket rtpPacket)
                 m_receptionReport = new ReceptionReport(rtpPacket.Header.SyncSource);
             }
 
-            m_receptionReport.RtpPacketReceived(rtpPacket.Header.SequenceNumber, rtpPacket.Header.Timestamp, DateTimeToNtpTimestamp32(DateTime.Now));
+            var arrivalTimestamp = PayloadSampleRateHz == 0 ? DateTimeToNtpTimestamp32(DateTime.Now) : (uint)((DateTime.Now - CreatedAt).TotalSeconds * PayloadSampleRateHz);
+            m_receptionReport.RtpPacketReceived(rtpPacket.Header.SequenceNumber, rtpPacket.Header.Timestamp, arrivalTimestamp);

```

Comment:
the relative transit time is the difference between a packet's RTP timestamp and the receiver's clock at the time of arrival, **measured in the same units.**
from https://datatracker.ietf.org/doc/html/rfc3550
arrivalTimestamp should be the same unit as rtpPacket.Header.Timestamp